src/rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date

```

Comment:
- [ ] Kindly add the `:book` and `:person` variables to the getter-setter here.

Reply:
not resolved